config/app.php

```diff
@@ -37,6 +37,9 @@
     /** ORCID */
     'ORCID_TOKEN' => env('ORCID_TOKEN', false),
 
+    /** enable scheduled orcid data update */
+    'enable_orcid_update' => env('ENABLE_ORCID_UPDATE', false),
```

Comment:
Can you also add this new `ENABLE_ORCID_UPDATE=false` to the `.env.example` file (along with a comment describing it)? It helps make the options more discoverable when someone sets up a new install.

Reply:
@wunc The requested change has been pushed to this branch in the last commit.